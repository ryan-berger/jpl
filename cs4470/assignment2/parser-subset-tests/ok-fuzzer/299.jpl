show 7
return j5J.(9537.45, F)
write image 6 to "NY~cj"
show .2896
show g()
return k6B_
time time show 37486292
assert f2m_(.43140), "-L"
read image "" to f1V.
print "l"
read image "" to M
write image 48 to "-"
write image a to ""
time time print ""
assert b8i_2u_4Y.1O_(), "g"
print "rG"
write image P6J.(N(), 8084, 53, b9d.9K.1M_, L()) to "c"
