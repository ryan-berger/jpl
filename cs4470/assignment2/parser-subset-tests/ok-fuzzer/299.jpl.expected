(ShowCmd (IntExpr 7))
(StmtCmd (ReturnStmt (CallExpr j5J. (FloatExpr 9537) (VarExpr F))))
(WriteImageCmd (IntExpr 6) "NY~cj")
(ShowCmd (FloatExpr 0))
(ShowCmd (CallExpr g))
(StmtCmd (ReturnStmt (VarExpr k6B_)))
(TimeCmd (TimeCmd (ShowCmd (IntExpr 37486292))))
(StmtCmd (AssertStmt (CallExpr f2m_ (FloatExpr 0)) "-L"))
(ReadImageCmd "" (VarArgument f1V.))
(PrintCmd "l")
(ReadImageCmd "" (VarArgument M))
(WriteImageCmd (IntExpr 48) "-")
(WriteImageCmd (VarExpr a) "")
(TimeCmd (TimeCmd (PrintCmd "")))
(StmtCmd (AssertStmt (CallExpr b8i_2u_4Y.1O_) "g"))
(PrintCmd "rG")
(WriteImageCmd
 (CallExpr
  P6J.
  (CallExpr N)
  (IntExpr 8084)
  (IntExpr 53)
  (VarExpr b9d.9K.1M_)
  (CallExpr L))
 "c")
